src/os/freertos/include/AbstractTask.tpp

```diff
@@ -35,8 +36,14 @@ bool AbstractTask<stackSize>::isRunning() {
 template <unsigned int stackSize>
 void AbstractTask<stackSize>::wrapper(void* params) {
     AbstractTask* task = static_cast<AbstractTask*>(params);
-    task->task();
-    task->m_taskRunning = false;
+
+    while (true) {
+        if (task->m_taskRunning) {
```

Comment:
I don't see it in the changes, but I guess there is already a function to stop the task that set m_taskRunning to false?

Reply:
Nope, is running is just to know internally if it should be running (thus  calling the user code), if not it's just suspends the task